doc/modules/calibration.rst

```diff
@@ -35,177 +42,117 @@ with different biases per method:
+
+The samples that are used to train the calibrator should not be used to
+train the target classifier.
+
+Usage
+-----
+
+The :class:`CalibratedClassifierCV` class is used to calibrate a classifier.
+
+:class:`CalibratedClassifierCV` uses a cross-validation approach to fit both
+the classifier and the regressor. For each of the k `(trainset, testset)`
+couple, a classifier is trained on the train set, and its predictions on the
+test set are used to fit a regressor. We end up with k
+`(classifier, regressor)` couples where each regressor maps the output of
+its corresponding classifier into [0, 1]. Each couple is exposed in the
+`calibrated_classifiers_` attribute, where each entry is a calibrated
+classifier with a :term:`predict_proba` method that outputs calibrated
+probabilities. The output of :term:`predict_proba` for the main
+:class:`CalibratedClassifierCV` instance corresponds to the average of the
+predicted probabilities of the `k` estimators in the
+`calibrated_classifiers_` list. The output of :term:`predict` is the class
+that has the highest probability.
+
+The regressor that is used for calibration depends on the `method`
+parameter. `'sigmoid'` corresponds to a parametric approach based on Platt's
+logistic model [3]_, i.e. :math:`p(y_i = 1 | f_i)` is modeled as
+:math:`\sigma(A f_i + B)` where :math:`\sigma` is the logistic function, and
+:math:`A` and :math:`B` are real numbers to be determined when fitting the
+regressor via maximum likelihood. `'isotonic'` will instead fit a
+non-parametric isotonic regressor, which outputs a step-wise non-decreasing
+function (see :mod:`sklearn.isotonic`).
+
+An already fitted classifier can be calibrated by setting `cv="prefit"`. In
+this case, the data is only used to fit the regressor. It is up to the user
+make sure that the data used for fitting the classifier is disjoint from the
+data used for fitting the regressor.
+
+:class:`CalibratedClassifierCV` can calibrate probabilities in a multiclass
+setting if the base estimator supports multiclass predictions. The classifier
+is calibrated first for each class separately in a one-vs-rest fashion [4]_.
+When predicting probabilities, the calibrated probabilities for each class
+are predicted separately. As those probabilities do not necessarily sum to
+one, a postprocessing is performed to normalize them.
+
+The :func:`sklearn.metrics.brier_score_loss` may be used to evaluate how
+well a classifier is calibrated.
+
+.. topic:: Examples:
+
+   * :ref:`sphx_glr_auto_examples_calibration_plot_calibration_curve.py`
+   * :ref:`sphx_glr_auto_examples_calibration_plot_calibration_multiclass.py`
+   * :ref:`sphx_glr_auto_examples_calibration_plot_calibration.py`
+   * :ref:`sphx_glr_auto_examples_calibration_plot_compare_calibration.py`
 
 .. topic:: References:
 
-    * Obtaining calibrated probability estimates from decision trees
-      and naive Bayesian classifiers, B. Zadrozny & C. Elkan, ICML 2001
```

Comment:
Isn't this one missing below?

Reply:
I removed it because I didn't see any relevant place where to link to this ref. I don't think we use the techniques described in this paper anywhere in the code, so adding it is just noise.

side note, @glemaitre suggested to unify the refs using the `.. [X] blahblah` format, and when using this format, all refs need to be linked to, otherwise our CI complains.